src/lib/MAVLink/MAVLink.cpp

```diff
@@ -99,6 +190,48 @@ void convert_mavlink_to_crsf_telem(uint8_t *CRSFinBuffer, uint8_t count, Handset
                 }
                 CRSF::SetHeaderAndCrc((uint8_t *)&crsffm, CRSF_FRAMETYPE_FLIGHT_MODE, CRSF_FRAME_SIZE(sizeof(crsffm)), CRSF_ADDRESS_CRSF_TRANSMITTER);
                 handset->sendTelemetryToTX((uint8_t *)&crsffm);
+
+                // send the ap_status message to Yaapu Telemetry Script
+                ap_send_crsf_passthrough_single(0x5001, format_ap_status(heartbeat.base_mode, heartbeat.custom_mode, heartbeat.system_status, throttle));
+                // send the frame_type parameter to Yaapu Telemetry Script
+                ap_send_crsf_passthrough_parameter(1, heartbeat.type);
+                break;
```

Comment:
It makes sense to swap these lines. set the parameter first and then send the ap_status to trigger.